glide.yaml

```diff
@@ -6,14 +6,13 @@ owners:
   email: rob@tigera.io
 import:
 - package: github.com/Sirupsen/logrus
-  version: ^0.10.0
```

Comment:
How about `>=0.10.0`?  `^1.0.0` conflicts with downstream projects that pin to `^0.10.0`